modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+= Backup and Restore
+
+The Backup and Restore functionality of the Kubernetes operator is robust, allowing for incremental and scheduled backups as well as detailed logging.
```

Comment:
```suggestion
The Backup and Restore functionality of the Kubernetes operator allows for incremental and scheduled backups as well as detailed logging.
```

Reply:
One should never qualify a backup feature as "robust" because a non-robust backup is very sketchy and should not exist in an enterprise product. If you really want to express some sense of strength, I would either say that we have a comprehensive set of features, or that our feature sets enable the user to deploy a robust backup strategy.  It would be their strategy that would be robust.